### PR TITLE
fixed imports, typing and returns

### PR DESCRIPTION
**Pull request by kmaphoenix**

### CODE REVIEW

File: src/dfcx_scrapi/core/environments.py [hunk 1]
```diff
@@ -32,13 +31,13 @@
...
-class Environments(ScrapiBase):
+class Environments(scrapi_base.ScrapiBase):
     """Core Class for CX Environments Resource functions."""
 
     def __init__(
         self,
         creds_path: str = None,
-        creds_dict: dict = None,
+        creds_dict: Dict[str,str] = None,
         creds=None,
```
Comment (Greenford) [inline src/dfcx_scrapi/core/environments.py:41]: @kmaphoenix type hint for creds for consistency

File: src/dfcx_scrapi/core/environments.py [hunk 2]
```diff
@@ -358,7 +357,7 @@ def lookup_environment_history(
             <Agent ID>/environments/<Environment ID>.
 
         Returns:
-          history: List of Environment objects with historical timestamps
+          List of Environment objects with historical timestamps
```
Comment (Greenford) [inline src/dfcx_scrapi/core/environments.py:360]: period? @kmaphoenix

File: src/dfcx_scrapi/core/environments.py [hunk 3]
```diff
@@ -387,7 +386,7 @@ def list_continuous_test_results(self, environment_id:str):
             <Agent ID>/environments/<Environment ID>.
 
         Returns:
-          test_results: List of types.ContinuousTestResult
+          List of types.ContinuousTestResult
```
Comment (Greenford) [inline src/dfcx_scrapi/core/environments.py:389]: period

File: src/dfcx_scrapi/tools/utterance_generator_util.py [hunk 4]
```diff
@@ -18,8 +18,8 @@
 import string
 import pandas as pd
 from typing import List
-from dfcx_scrapi.core.scrapi_base import ScrapiBase
-from dfcx_scrapi.core.intents import Intents
+from dfcx_scrapi.core import scrapi_base
+from dfcx_scrapi.core import intents
 from dfcx_scrapi.core_ml.utterance_generator import UtteranceGenerator
```
Comment (Greenford) [inline src/dfcx_scrapi/tools/utterance_generator_util.py:23]: @kmaphoenix  should be:
`from dfcx_scrapi.core_ml import utterance_generator `

File: src/dfcx_scrapi/core/environments.py (hunk 1 above)
Comment (kmaphoenix) [reply src/dfcx_scrapi/core/environments.py:41]: ack.

File: src/dfcx_scrapi/core/environments.py (hunk 2 above)
Comment (kmaphoenix) [reply src/dfcx_scrapi/core/environments.py:360]: ack

File: src/dfcx_scrapi/core/environments.py (hunk 3 above)
Comment (kmaphoenix) [reply src/dfcx_scrapi/core/environments.py:389]: ack

**Review by Greenford — CHANGES_REQUESTED** [review]
@kmaphoenix commented and tagged you - just a few docstring returns needing a period at the end, an import change for the utterance generator. Otherwise LGTM!

File: src/dfcx_scrapi/tools/utterance_generator_util.py (hunk 4 above)
Comment (kmaphoenix) [reply src/dfcx_scrapi/tools/utterance_generator_util.py]: ack, updated.

**Review by Greenford — APPROVED** [review]

**Pull request merged**